scripts/chain-prep.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#set -eux
+killall node
+ganache-cli -u 0 -u 1 -u 2 -m 'cook mango twist then skin sort option civil have still rather guilt' &
+git clone https://github.com/althea-mesh/simple-bidirectional-erc20-channel contract
+pushd contract 
+npm install .
+truffle compile
+addresses="$(truffle migrate --verbose)"
+export channel_address="$(echo $addresses | grep -Eo ': 0x[0-9a-f]{40}'| cut -d$'\n' -f3)"
```

Comment:
Above there is a a script execution "bash chain-prep.sh" - those exported variables wouldn't be visible to other child processes after the bash command is done. Instead of "bash chain-prep.sh" we could do "source chain-prep.sh" or ". ./foo.sh". (or just echo "export" commands to stdout and `eval "$(bash chain-pre.sh)"` if we want to be fancy).

As a side note what do you think of upper case env var names?

Reply:
I was actually debating with myself if we should upper case them or not. I guess I will for convention. You're right about the sourcing too.